locale/lang-tools/add_string.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+import argparse
+import toml
+from glob import glob
+from merge import unmerge
+
+def copy_template_internal(res, template):
+    for key in template:
+        if key not in res and key != 'ALERTS':
+            res[key] = '~~'+ template[key]
```

Comment:
I also preferred the old
 empty_string + //comment_with_phrase_to_translate

The comment  with the phrase to translate is important and should be kept in every file